server/index.js

```diff
@@ -12,6 +12,9 @@ db.on('error', console.error.bind(console, 'error'))
 db.once('open', () => {
   console.log('Conectado satisfactoriamente a la base de datos muebleria')
 })
+const router = require('./router')
+
+app.use('/', router)
 
 app.use(bodyParser.urlencoded({ extended: false}))
 app.use('/', rutas)
```

Comment:
this is not your change, but i think is a good idea to remove this app.use cause we are duplicating the routes, let's just keep the one that is on line 17 and delete the other one